bioconda_utils/utils.py

```diff
@@ -289,72 +295,41 @@ def build(recipe,
     if not force:
         build_args += ["--skip-existing"]
 
+    logger.debug('config: %s', config)
     channel_args = []
-    for c in config['channels']:
+    for c in config.get('channels', []):
         channel_args.extend(['--channel', c])
 
-    if docker is not None:
-        conda_build_folder = os.path.join(os.path.dirname(os.path.dirname(shutil.which("conda"))), "conda-bld")
-        logger.info("Using client's conda-build folder: %s", conda_build_folder)
-        recipe = os.path.relpath(recipe, recipe_folder)
-        binds={
-            pkg_resources.resource_filename('bioconda_utils', 'bioconda_startup.sh'): {
-                "bind": "/opt/share/bioconda_startup.sh",
-                "mode": "ro"
-            },
-            os.path.abspath(recipe_folder): {
-                "bind": "/opt/recipes",
-                "mode": "ro"
-            },
-        }
-        for subdir in ['svn-cache', 'hg-cache', 'git-cache', 'src-cache', 'linux-64']:
-            binds[os.path.join(conda_build_folder, subdir)] = {
-                'bind': os.path.join('/opt/miniconda/conda-bld/', subdir),
-                'mode': 'rw'
-            }
-        env = dict(env)
-        env["ABI"] = 4
-        logger.debug('Docker binds: %s', binds)
-        logger.debug('Docker env: %s', env)
-
-        user_id = 1000  #os.getuid()
-        command = ("bash /opt/share/bioconda_startup.sh {uid} "
-                "conda build {args} --override-channels --quiet recipes/{recipe}".format(
-                    uid=user_id, args=' '.join(channel_args), recipe=recipe))
-        logger.debug('Docker command: %s', command)
-        container = docker.create_container(
-            image=config['docker_image'],
-            volumes=["/opt/recipes", "/opt/miniconda"],
-            user=user_id,
-            environment=env,
-            command=command,
-            host_config=docker.create_host_config(binds=binds, network_mode="host"))
-        cid = container["Id"]
-        docker.start(container=cid)
-        status = docker.wait(container=cid)
-        logger.debug('Docker status: %s', status)
-        if status != 0:
-            logger.error(docker.logs(container=cid, stdout=True, stderr=True).decode())
-            return False
-        logger.info('Successfully built %s', recipe)
-        return True
-    else:
-        try:
+    CONDA_BUILD_CMD = ['conda', 'build', '--quiet', '--override-channels'] + channel_args + build_args
```

Comment:
Thanks, this was leftover from testing.

`--override-channels` and other args to conda-build are passed in at container run-time in the `build_recipe` method.